sdcm/nemesis.py

```diff
@@ -4784,8 +4784,10 @@ def finalizer(exc_type, *_):
                         self.cluster.decommission(new_node)
             context_manager.push(finalizer)
 
-            with temporary_replication_strategy_setter(node) as replication_strategy_setter:
+            with self.run_nemesis(node_list=self.cluster.data_nodes, nemesis_label=self.current_disruption) as node, \
+                    temporary_replication_strategy_setter(node) as replication_strategy_setter:
                 new_node = self._add_new_node_in_new_dc()
+                self.set_current_disruption(new_node)
```

Comment:
This is was added by https://github.com/scylladb/scylla-cluster-tests/pull/9425. Please, rebase